office-actions/src/automation/excel/excel.action-handler.ts

```diff
@@ -343,29 +343,31 @@ export default class ExcelActionHandler extends StatefulActionHandler {
         if (input.worksheet) this.checkIsWorksheetNameCorrect(input.worksheet, true);
 
         const targetWorksheet = this.session.Worksheets(input?.worksheet ?? this.session.ActiveSheet.Name);
-
         const rowsRange = targetWorksheet.Range(input.cellRange).Rows;
         const rowsCount = rowsRange.Count;
 
         if (input.rowLevel && Number(input.rowLevel) <= 0) {
             throw new Error(ExcelErrorMessage.createHtmlTableInvalidRowLevel());
         }
-        const rowLevel = input.rowLevel ? Number(input.rowLevel) : 1;
```

Comment:
why?

Reply:
cause i do not use this variable anymore